docs-ref-conceptual/spring-framework/configure-spring-boot-java-applicationinsights.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Configure a Spring Boot Initializer app to use Azure Application Insights SpringBoot Starter
+description: Configure a Spring Boot application created with the Spring Initializr to use Appliaction Insights SpringBoot Starter.
+services: Application-Insights
+documentationcenter: java
+author: dhdoshi
+manager: alexklim
+editor: ''
+
+ms.assetid:
+ms.author: dhdoshi
+ms.date: 05/19/2018
+ms.devlang: java
+ms.service: Azure Monitor
+ms.tgt_pltfrm: application-insights
+ms.topic: article
+ms.workload: na
+---
+
+# Configure a Spring Boot Initializer app to use Application Insights with Azure Application Insights Spring Boot Starter (PUBLIC PREVIEW)
+
+This article walks you through creating a Spring Boot application using **[Spring Initializr]**, that uses Azure Application Insights Spring Boot Starter for end-to-end monitoring of Java applications on cloud. *Please note this starter is currently in **BETA (public preview)**.*
+
+## Prerequisites
+
+The following prerequisites are required in order to complete the steps in this article:
+
+* An Azure subscription; if you don't already have an Azure subscription, you can activate your [MSDN subscriber benefits] or sign up for a [free Azure account].
+* A [Java Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/), version 1.7 or later.
+* [Apache Maven](http://maven.apache.org/), version 3.0 or later.
+
+## Create a custom application using the Spring Initializr
+
+1. Browse to <https://start.spring.io/>.
+
+1. Specify that you want to generate a **Maven** project with **Java**, enter the **Group** and **Aritifact** names for your application, and then select web dependency in the dependenies section.
```

Comment:
Spelling: 
change **Aritifact** to **Artifact**
change dependenies to dependencies